src/object/srv_obj.c

```diff
@@ -1085,6 +1164,9 @@ obj_local_rw(crt_rpc_t *rpc, struct ds_cont_hdl *cont_hdl,
 		}
 	}
 
+	if (obj_rpc_is_fetch(rpc))
```

Comment:
iom is option right? if user don't need it (pass NULL iom ptr for obj_fetch) then need not create and return it, as it will add overhead for each fetch.

Reply:
Correct. Added a flag to whether or not iom should be created. Other stories will use that flag when appropriate.